optaplanner-examples/src/main/java/org/optaplanner/examples/flightcrewscheduling/domain/Employee.java

```diff
@@ -66,6 +66,8 @@ public boolean isLastAssignmentArrivingAtHome() {
     }
 
     public long countInvalidConnections() {
+        // TODO Cache this to improve example performance.
+        //  Especially useful for Constraint Streams, where this is called multiple times per Employee.
```

Comment:
Code style: 2 spaces before the E. No space in ConstraintStreams. "Where" -> "by which"